tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -106,13 +106,13 @@ Given('user {string} has uploaded file {string} to {string}', async function(
 })
 
 When('the user browses to display the {string} details of file {string}', async function(
-  versions,
+  tab,
   filename
 ) {
   const api = client.page.FilesPageElement
   await api.filesList().clickRow(filename)
   await client.initAjaxCounters()
-  await api.appSideBar().openVersionsTab()
+  await api.appSideBar().selectTab(tab)
```

Comment:
this was never correct, but so far this step was only used for versions, so we never noticed